src/test/java/core/basesyntax/service/impl/WriteServiceImplTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class WriteServiceImplTest {
+    private static final String PATH_TO_FILE = "src/test/resources/report.csv";
+    private static final String INVALID_PATH = "invalid/path/to/file";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static WriteServiceImpl writeService;
+
+    @BeforeAll
+    public static void setUp() {
+        writeService = new WriteServiceImpl();
+    }
+
+    @Test
+    public void writeToFile_Ok() throws IOException {
+        String expectedReport = "fruit,quantity" + LINE_SEPARATOR
+                + "apple,20" + LINE_SEPARATOR
+                + "banana,50" + LINE_SEPARATOR
+                + "orange,30" + LINE_SEPARATOR;
+        writeService.writeToFile(PATH_TO_FILE, expectedReport);
+        List<String> expectedLines = Arrays.asList(expectedReport.split(LINE_SEPARATOR));
+        List<String> actualLines = Files.readAllLines(Path.of(PATH_TO_FILE));
+        assertEquals(expectedLines.size(), expectedLines.size());
+        for (int i = 0; i < expectedLines.size(); i++) {
+            assertEquals(expectedLines.get(i), actualLines.get(i));
+        }
+    }
+
+    @Test
+    public void writeToFile_InvalidPath_NotOk() {
```

Comment:
follow name convention - word after underscore should start from lower case letter 
fix it for all tests 
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/32339836/9f0bea6c-1153-4a8e-91f9-d0d7b6afc792)